### DISCUSSION TITLE
Iteration

### DISCUSSION THREAD

**Discussion by TenFont** [SkriptLang/Skript#4018] [Suggestions]
A way of executing the same line of code multiple times in a single line, without using long loops.
```hs
iterate through all players to set {balance::%element%} to 0
```

**Comment by ghost** [comment]
i agree about this one but dont you think its a little hard to write all of that when you could do 
`loop all players to set {balance::%loop-player%} to 0`

**Reply by abandonedaccount6235** [reply]
`set {balance::%loop-player%} to 0 for each player in all players`
ideally the for each would be at the start but i don't know if the parser can handle that without a word between the expr and the eff

**Comment by AyhamAl-Ali** [comment]
Great idea I would love to see in Skript, would be a lot useful in Skript's effect commands as well
